mappings/net/minecraft/util/math/Direction.mapping

```diff
@@ -28,6 +29,12 @@ CLASS net/minecraft/class_2350 net/minecraft/util/math/Direction
 	METHOD method_10143 byId (I)Lnet/minecraft/class_2350;
 		ARG 0 id
 	METHOD method_10144 asRotation ()F
+	METHOD method_10145 toDirectionArray (Lnet/minecraft/class_2350;Lnet/minecraft/class_2350;Lnet/minecraft/class_2350;)[Lnet/minecraft/class_2350;
```

Comment:
this is like listing the closest directions, used by entity facing order i think. Name needs improvement

Reply:
Maybe `listClosestDirections`?
```suggestion
	METHOD method_10145 listClosestDirections (Lnet/minecraft/class_2350;Lnet/minecraft/class_2350;Lnet/minecraft/class_2350;)[Lnet/minecraft/class_2350;
```
the first, second, third names can be retained.